packages/@react-spectrum/button/test/ButtonGroup.test.js

```diff
@@ -97,7 +179,44 @@ describe('ButtonGroup', function () {
     expect(group).toHaveAttribute('aria-disabled', 'true');
   });
 
-  it('SideNav handles single selection', function () {
+  it.each`
+    Name                                                   | ComponentGroup   | Component       | props                                         | orders
+    ${'(left/right arrows, ltr + horizontal) ButtonGroup'} | ${ButtonGroup}   | ${ActionButton} | ${{locale: 'de-DE'}}                          | ${[{action: pressArrowRight, result: btnBehavior.forward}, {action: pressArrowLeft, result: btnBehavior.backward}, {action: pressArrowLeft, result: btnBehavior.backward}]}
+    ${'(left/right arrows, rtl + horizontal) ButtonGroup'} | ${ButtonGroup}   | ${ActionButton} | ${{locale: 'ar-AE'}}                          | ${[{action: pressArrowRight, result: btnBehavior.forward}, {action: pressArrowLeft, result: btnBehavior.backward}, {action: pressArrowLeft, result: btnBehavior.backward}]}
```

Comment:
Is this really the right behavior? I feel like i expect and see:
ltr: right -> forward, left -> back
rtl: right -> back, left -> forward

Reply:
actually, I don't see any differences between rtl and ltr actually

Reply:
ah whoops, I hard coded the locale, will fix